service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/respondentsolicitor/UnregisteredRespondentSolicitorContentProviderTest.java

```diff
@@ -1,40 +1,102 @@
 package uk.gov.hmcts.reform.fpl.service.email.content.respondentsolicitor;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
 import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
 import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
 import uk.gov.hmcts.reform.fpl.model.notify.respondentsolicitor.UnregisteredRespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.config.LookupTestConfig;
+import uk.gov.hmcts.reform.fpl.service.email.content.AbstractEmailContentProviderTest;
+import uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper;
 
+import java.util.List;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 
-class UnregisteredRespondentSolicitorContentProviderTest {
+@ContextConfiguration(classes = {UnregisteredRespondentSolicitorContentProvider.class, LookupTestConfig.class,
+    EmailNotificationHelper.class})
```

Comment:
notification helper and lookup test config shouldn't be needed here as you are providing them as mock beans below

Reply:
removed the unused beans